packages/components/src/components/post-popover/post-popover.scss

```diff
@@ -21,8 +21,7 @@
   align-items: self-start;
   padding: 0.5em;
 
-  // Max 2/3 of the column width
-  max-width: min(66vw, 60rem);
+  max-width: 280px;
```

Comment:
The Figma design includes a min- and a max-width. Why was this removed?
https://www.figma.com/file/xZ0IW0MJO0vnFicmrHiKaY/Components-Post?type=design&node-id=18199-9350&mode=design&t=1TuHz3HP8n0aZsF9-0

Reply:
Because with these dynamically value, you don't get the same result as the design. If you look at the current version. It makes the popover much longer than in the design and the readability is poor.